spec/models/user_spec.rb

```diff
@@ -33,6 +33,13 @@
     expect(another_user.roles[1]).to eq(cfp_role)
   end
 
+  describe '#name' do
+    it 'returns the username as name if there is not name' do
```

Comment:
It would nice if we have test for case where it `returns the name if name is present`.